modules/apps/commerce/commerce-cart-content-web/src/main/resources/META-INF/resources/js/cart_mini/transition.js

```diff
@@ -13,87 +13,93 @@
  */
 
 export default function ({namespace}) {
-	Liferay.provide(window, `${namespace}transition`, (event) => {
-		const link = event.delegateTarget;
+	Liferay.provide(
+		window,
+		`${namespace}transition`,
+		(event) => {
+			const link = event.target.closest('.transition-link');
 
-		const workflowTaskId = parseInt(link.dataset.workflowtaskid, 10);
+			const workflowTaskId = parseInt(link.dataset.workflowtaskid, 10);
 
-		const form = document.getElementById(`${namespace}transitionFm`);
+			const form = document.getElementById(`${namespace}transitionFm`);
 
-		document.getElementById(`${namespace}transitionCommerceOrderId`).value =
-			link.dataset.commerceorderid;
+			document.getElementById(
+				`${namespace}transitionCommerceOrderId`
+			).value = link.dataset.commerceorderid;
```

Comment:
Extract the doc.getelbyid into its own variable and check if it exists before assigning it the link.dataset.commerceorderid.

Reply:
Done as requested.